### DISCUSSION TITLE
Have errors : 2 in random parts of g code

### DISCUSSION THREAD

**Discussion by MadMaker21** [fra589/grbl-Mega-5X#269] [Q&A]
Some of programs works perfectly, but some have this problem, is anyone have this or fix it already? I heard something about this problem, someone said like it's eeprom errors. If I change the program, add or delete some parts of code it change place of stopping the code.

**Comment by fra589** [comment]
Il faudrait un peu plus d'information pour analyser le problème...
Quelle est ta version de grbl-Mega-5X ?
Quelle est la configuration des axes ?
Quels paramètres $$ ?
Quel GCode ?
Bref, le fait de dire simplement "j'ai une erreur" ne suffit vraiment pas à te répondre...
